src/tests/unit/tests/popup/incompatible-browser-renderer.test.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import ReactDOM from 'react-dom';
+import { It, Mock } from 'typemoq';
+
+import { IncompatibleBrowserRenderer } from '../../../../popup/incompatible-browser-renderer';
+
+describe('IncompatibleBrowserRenderer', () => {
+    it('render', () => {
+        const renderMock = Mock.ofType<typeof ReactDOM.render>();
+        const containerStub = Mock.ofType<HTMLElement>();
```

Comment:
Since this is a mock, should it be called `containerMock`?

Reply:
I'm not using any mock capabilities with it so that's why I called it that; I can change it to mock.